test/integration/cloudresource_e2e_test.go

```diff
@@ -198,6 +199,29 @@ var _ = Describe(fmt.Sprintf("%s,%s: NetworkPolicy On Cloud Resources", focusAws
 			}
 			err := utils.ConfigureK8s(kubeCtl, defaultANPParameters, k8stemplates.DefaultANPSetup, false)
 			Expect(err).ToNot(HaveOccurred(), "failed to add default ANP rule")
+
+			// Check antrea-agent has connected to antrea-controller.
+			for _, vpc := range cloudVPCs {
+				prefix := "virtualmachine"
```

Comment:
Why create a variable when its used only once

Reply:
Sure. Will remove it.

Reply:
Done